risc0/zkvm/methods/guest/src/bin/random_number.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 RISC Zero, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![no_std]
+#![no_main]
+
+use risc0_zkvm::guest::env;
+use risc0_zkvm_platform::syscall::sys_rand;
+
+risc0_zkvm::entry!(main);
+
+pub fn main() {
+    unsafe {
+        let rbuf = sys_rand(5);
+        env::commit_slice(&rbuf);
+        // assert_eq!(rbuf, [0u8; 10])
```

Comment:
I think we should add this to the existing multi_test binary.

Reply:
ah, I forgot about that, thanks!